### PR TITLE
Reduce systest log level in CI

### PR DESCRIPTION
**Pull request by poszu**
## Motivation

Closes #

## TODO

- [ ] Explain motivation or link existing issue(s)
- [ ] Test changes and document test plan
- [ ] Update documentation as needed

## DevOps Notes

- [x] This PR does not require configuration changes (e.g., environment variables, GitHub secrets, VM resources)
- [x] This PR does not affect public APIs
- [x] This PR does not rely on a new version of external services (PoET, elasticsearch, etc.)
- [x] This PR does not make changes to log messages (which monitoring infrastructure may rely on)

### CODE REVIEW

**Comment by poszu** [comment]
bors try

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -338,7 +338,7 @@ jobs:
           node_selector: cloud.google.com/gke-nodepool=gha
           size: 30
           bootstrap: 10m
-          level: debug
+          level: info
```
Comment (countvonzero) [inline .github/workflows/ci.yml:341]: can you keep this as debug please.

Comment (poszu) [reply .github/workflows/ci.yml:341]: I want to check if reducing log level helps with the timeouts.

Comment (countvonzero) [reply .github/workflows/ci.yml:341]: actually never mind. i am fine with info for this test.

**Comment by poszu** [comment]
Bors try

**Comment by poszu** [comment]
Bors try

**Comment by poszu** [comment]
bors try

**Comment by poszu** [comment]
Duplicate of #3892

**Pull request closed without merging**